libraries/core_libs/storage/src/storage.cpp

```diff
@@ -419,6 +419,10 @@ void DbStorage::clearPeriodDataHistory(uint64_t period) {
   db_->DeleteRange(write_options_, handle(Columns::period_data), toSlice(start), toSlice(period));
 }
 
+void DbStorage::overridePeriodData(const SyncBlock& sync_block) {
```

Comment:
I dislike the idea of overwriting period data. I think the same could possibly be accomplished by saving additional cert votes in the next block

Reply:
The reward votes (only hash) in period N PBFT block, that's exactly same as cert votes (object) in period N-1 PBFT block.
One cert vote vote on block B1, cannot save the cert votes in next block B2

Reply:
Miho propose a new idea. Add additional cert votes in next block, instead of overwriting previous period data in DB.
We can discuss the idea in our meeting